index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+
+    <h1 id="title">Survey Form</h1>
+
+    <p id="description">
+      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Qui
+      exercitationem dolore natus, earum, nesciunt ad consequuntur corporis
+      repellendus nisi, eligendi quos voluptatibus harum iusto dignissimos? Sed
+      deserunt omnis enim dolore.
+    </p>
```

Comment:
- Please when requiring a change from a student, try to explain why the change is required so that the student won't get confused or misunderstand you.
- Please use inline comments as much as possible, so that the student knows where you are pointing to.
- [Optional] You can use the list feature instead of using an `#`

**Tip**
- Please try give at least three inline comments review on this project, so as to make it approved, and follow the guidelines I gave above.